requirements.in

```diff
@@ -29,6 +29,7 @@ elasticsearch==6.3.0
 factory_boy
 faker
 feedparser
+google-api-python-client
```

Comment:
Can you pin this to a specific version (e.g. `google-api-python-client==1.7.11`) so we don't accidentally get an upgrade that might break a deploy or branch?